tests/terraform/eks_nodegroup_delete/main.tf

```diff
@@ -0,0 +1,155 @@
+provider "aws" {
+  region = "eu-central-1"
```

Comment:
we don't want region hard coded here, we want this to flow from environment variables so the test runner can target different regions, terraform uses AWS_REGION